superset-frontend/src/views/CRUD/welcome/ChartTable.tsx

```diff
@@ -86,6 +91,13 @@ function ChartTable({
 
   const [chartFilter, setChartFilter] = useState('Mine');
 
+  useEffect(() => {
+    const filter = getFromLocalStorage('chart', null);
```

Comment:
I think we should use more specific names for the local storage keys than `activity` and `chart` since these names need to be shared globally across the entire application. Perhaps use `homepage_chart_filter` here.